scripts/finetune.py

```diff
@@ -182,15 +182,18 @@ def train(
         if cfg.bf16:
             cfg.fp16 = True
         cfg.bf16 = False
+        
+    # Store inference mode into cfg when passed via args
+    cfg.inference = True if "inference" in kwargs else cfg.get("inference", False)
```

Comment:
I don't think you need to do this as the following should've covered it https://github.com/OpenAccess-AI-Collective/axolotl/blob/febe902517b11e5280b4325530ddc26fd3553694/scripts/finetune.py#L149-L159

Reply:
Indeed. Tested and this works (cfg.strict==False)

The only difference without this line is cfg.inference == None instead of False when specified neither in the config nor in the args.
No consequence if tested with "not", but not formally correct.

This also hints that the 2 other bool switches tested via "in kwargs" could be handled as cfg. as well, to harmonize everything.

Reply:
Hey

> No consequence if tested with "not", but not formally correct.

Yes, that would be fine, I think. The code base right now uses a lot of these, although not formally correct as you've said.

> This also hints that the 2 other bool switches tested via "in kwargs" could be handled as cfg. as well, to harmonize everything.

Uhm, for this PR, I would try to keep things small as possible. Also, those kwargs are not used elsewhere, so it's ok.